test/conformance/ingress/util.go

```diff
@@ -1046,7 +1046,16 @@ func CreateDialContext(ctx context.Context, t *testing.T, ing *v1alpha1.Ingress,
 	// TODO(mattmoor): I'm open to tricks that would let us cleanly test multiple
 	// public load balancers or LBs with multiple ingresses (below), but want to
 	// keep our simple tests simple, thus the [0]s...
-
+	dial := network.NewBackoffDialer(dialBackoff)
+	ingressIP := ing.Status.PublicLoadBalancer.Ingress[0].IP
+	if ingressIP != "" {
+		return func(ctx context.Context, _ string, address string) (net.Conn, error) {
+			if ingressIP != "" {
+				return dial(ctx, "tcp", ingressIP+":80")
+			}
+			return nil, errors.New("service ingress does not contain dialing information")
+		}
+	}
```

Comment:
this change seems out of scope for the PR description

Reply:
this was added by mistake as part of my local testing, took it out